api-aws-lambda/src/main/kotlin/org/coepi/api/v4/TCNCloudAPIHandler.kt

```diff
@@ -53,13 +65,11 @@ class TCNCloudAPIHandler: RequestHandler<APIGatewayProxyRequestEvent, APIGateway
         var body: String?
 
         try {
-            val queryParams = parseQueryParameters(input)
+            val (maybeDate, maybeInterval) = parseQueryParameters(input)
```

Comment:
This can be moved to the core along with the entire parseQueryParameters logic. parseQueryParameters can be changed to accept queryparams map directly instead of taking this object.